guides/common/modules/proc_increasing-the-logging-level-for-puppet-server.adoc

```diff
@@ -0,0 +1,18 @@
+:_mod-docs-content-type: PROCEDURE
+
+[id="increasing-the-logging-level-for-puppet-server"]
+= Increasing the logging level for Puppet server
+
+You can find the logs in `/var/log/puppetlabs/puppetserver/`.
+
+.Prerequisites
+* You have enabled Puppet on your {Project}.
+For more information, see {ManagingConfigurationsPuppetDocURL}Enabling_Puppet_Integration_managing-configurations-puppet[Enabling Puppet Integration with {Project}] in _{ManagingConfigurationsPuppetDocTitle}_.
```

Comment:
```suggestion
ifdef::satellite[]
.Prerequisites
* You have enabled Puppet on your {Project}.
For more information, see {ManagingConfigurationsPuppetDocURL}Enabling_Puppet_Integration_managing-configurations-puppet[Enabling Puppet Integration with {Project}] in _{ManagingConfigurationsPuppetDocTitle}_.
endif::[]
```
Enabled by default in Foreman?

Reply:
It's still a prerequisite and users can disable the integration so I'd keep it as is.

Note that katello (and probably orcharhino) also disable it out of the box.